wazuh/wazuh-dashboard-plugins
# Change vulnerabilities implicit filter field to wazuh.cluster.name

### Description
This pull request changes the implicit vulnerabilities filter field to wazuh.cluster.name whether the selected Wazuh server has cluster mode enabled or not.
 
### Issues Resolved
Closes #6650 

### Evidence

**Wazuh server cluster configuration**
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/74b8a941-95da-4c4c-a52c-9ff8030842d7)


**Single node mode implicit filter**
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/9343732/f1ebfa3f-0f76-40b6-8e15-3595313633f7)

### Test
- Configure the Wazuh server to have cluster mode `enabled`
  - Go to Vulnerabilities **dashboard** and check the implicit filter uses `wazuh.cluster.name`
  - Go to Vulnerabilities **inventory** and check the implicit filter uses `wazuh.cluster.name`
  - Go to Vulnerabilities **events** and check the implicit filter uses `cluster.name` 

- Configure the Wazuh server to have cluster mode `disabled`
  - Go to Vulnerabilities **dashboard** and check the implicit filter uses `wazuh.cluster.name`
  - Go to Vulnerabilities **inventory** and check the implicit filter uses `wazuh.cluster.name`
  - Go to Vulnerabilities **events** and check the implicit filter uses `manager.name` 


### Check List
